src/vscode-bicep/package.json

```diff
@@ -51,11 +51,11 @@
     "onCommand:bicep.gettingStarted.copyToClipboard",
     "onCommand:bicep.gettingStarted.createBicepFile",
     "onCommand:bicep.gettingStarted.openBicepFile",
-    "onWebviewPanel:bicep.visualizer"
+    "onWebviewPanel:bicep.visualizer",
```

Comment:
Did you mean to include this file with the check-in?

Reply:
Sorry that was unintentional